lib/authlogic/acts_as_authentic/login.rb

```diff
@@ -152,10 +152,14 @@ def find_by_smart_case_login_field(login)
         private
 
           def find_with_case(field, value, sensitivity = true)
+            gem_version = Gem::Version.new(Rails.version)
```

Comment:
Yes sure, seems to me that we should remove Rails.version and only use ActiveRecord::VERSION::STRING in this method, shouldn't we?